plugins/modules/source_control/github/github_repo.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Álvaro Torres Cogollo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from github import Github, GithubException
+import sys
+
+DOCUMENTATION = r'''
+---
+module: github_repo
+short_description: Manage your repos on Github
+description:
+- Manages Github repos using PyGithub library
+- Authentication can be done with access_token or with username and password
+options:
+  username:
+    description:
+    - Username parameter for authentication.
+    - This is only needed when not using access_token.
```

Comment:
```suggestion
    - This is only needed when not using I(access_token).
```
Use `I(...)` to reference module option names (see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for an overview of all formatting).